whitepapers/0013_ccq.md

```diff
@@ -289,6 +295,35 @@ Currently the only supported query types are `eth_call` and `eth_call_by_timesta
    []byte   batch_call_data
    ```
 
+#### Solana Queries
+
+Currently the only supported query type on Solana is `sol_account`.
+
+1. sol_account (query type 10) - this query is used to read data for one or more accounts on Solana.
```

Comment:
Just curious, why `10`?

Reply:
Thought I'd leave some space for additional eth ones.